sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/models/_models.py

```diff
@@ -1267,9 +1223,9 @@ class FollowupRecommendationInference(RadiologyInsightsInference, discriminator=
     :vartype extension: list[~azure.healthinsights.radiologyinsights.models.Extension]
     :ivar kind: Inference type. Required. Recommendation inference type
     :vartype kind: str or ~azure.healthinsights.radiologyinsights.models.FOLLOWUP_RECOMMENDATION
-    :ivar effective_date_time: Date and time are displayed when the procedure is recommended to be
-     done at a specific point in time.
-    :vartype effective_date_time: str
+    :ivar effective_at: Date and time are displayed when the procedure is recommended to be done at
```

Comment:
I believe these renames are missing in the changelog, same for the one on line 1199